src/js/pages/room/index.js

```diff
@@ -529,10 +529,37 @@ const Settings = ({open}) => {
   // Hooks
   //
   const [screen, setScreen] = useState(SETTINGS_SCREEN_MENU);
+  const [codeCopied, setCodeCopied] = useState(false);
+  const inviteCode = useSelector(BuoySelectors.connectedBuoy).get('token');
```

Comment:
might want to be a bit defensive here, as there's a chance the `RoomPage` could be loaded while also not being connected to a buoy (which would cause this to crash, since `BuoySelectors.connectedBuoy` could return null). Maybe something like:

```
const connectedBuoy = useSelector(BuoySelectors.connectedBuoy);
const inviteCode = connectedBuoy ? connectedBuoy.get('token') : '';
```